packages/manager/modules/pci/src/projects/project/storages/databases/database/advanced-configuration/advanced-configuration.html

```diff
@@ -0,0 +1,155 @@
+                                                ng-if="$ctrl.model[$index].key.values"
+                                                data-model="$ctrl.model[$index].value"
+                                                data-items="$ctrl.model[$index].key.values"
+                                                class="oui-select__input"
+                                            >
+                                            </oui-select>
+                                        </oui-field>
+                                    </div>
+                                </div>
+                            </fieldset>
+                            <footer
+                                class="oui-inline-adder__footer"
+                                ng-if="!$ctrl.model[$index].isFixed"
+                            >
+                                <button
+                                    type="button"
+                                    class="oui-inline-adder__action oui-button oui-button_primary oui-button_s"
+                                    data-ng-click="$ctrl.onAddConfig($index)"
+                                    data-ng-if="!$ctrl.model[$index].added"
+                                    data-ng-disabled="!$ctrl.model[$index].key || $ctrl.addableProperties.length <= 1 || $ctrl.pending"
+                                >
+                                    <span class="oui-icon oui-icon-add"></span>
+                                </button>
+                                <button
+                                    data-ng-disabled="$ctrl.pending"
+                                    type="button"
+                                    class="oui-inline-adder__action oui-button oui-button_primary oui-button_s"
+                                    data-ng-click="$ctrl.onRemoveConfig($index)"
+                                    data-ng-if="$ctrl.model[$index].added"
+                                >
+                                    <span
+                                        class="oui-icon oui-icon-trash"
+                                    ></span>
+                                </button>
+                            </footer>
+                        </div>
+                    </div>
+                </div>
+            </form>
+        </div>
+        <div class="col-lg-6">
+            <div class="sticky-top">
+                <!-- error messages -->
+                <cui-message-container
+                    data-messages="$ctrl.messages"
+                ></cui-message-container>
+                <pre class="oui-background-p-800 oui-color-p-050">
+{{$ctrl.getAdvancedConfigModel() | json}}</pre
+                >
+                <oui-button
+                    class="float-right"
+                    type="submit"
+                    size="l"
+                    disabled="$ctrl.pending"
+                    variant="primary"
+                    ng-click="$ctrl.updateAdvancedConfiguration()"
```

Comment:
For consistency:
```suggestion
                    on-click="$ctrl.updateAdvancedConfiguration()"
```

Reply:
Changed: https://github.com/ovh/manager/pull/6568/commits/d41de0c46bc372d3834ffe66082fcb7eca3aa01d